### PR TITLE
Android - React Native 0.54.1 fix

### PR DESCRIPTION
**Pull request by andreiciceu**
Based on the solution from @insidelabs

### CODE REVIEW

File: src/CircularProgress.js [hunk 1]
```diff
@@ -33,10 +26,10 @@ export default class CircularProgress extends React.Component {
 
   render() {
     const { size, width, tintColor, backgroundColor, style, rotation, linecap, children } = this.props;
-    const backgroundPath = this.circlePath(size / 2, size / 2, size / 2 - width / 2, 0, 360);
+    const backgroundPath = this.circlePath(size / 2, size / 2, size / 2 - width / 2, 0, 360 * 99.9 / 100);
```
Comment (Jacse) [inline src/CircularProgress.js]: How come we need to make it `359.99` instead of `360`?

Comment (Jacse) [inline src/CircularProgress.js]: You have used `* .99` later in the same file, here you use `* 99.9 / 100`. If we really need to do this, even though it feels hacky to me, we should stick to either and not use both.

Comment (andreiciceu) [inline src/CircularProgress.js]: Leaving it to 360 with 100% fill, does not render.

Comment (Jacse) [inline src/CircularProgress.js]: @andreiciceu Makes sense. Is this only a problem on Android, though? Question is whether we should do `.999` on Android and `360` on iOS. I think doing the same thing on both platforms, as with your PR, is the best approach. That way the library performs in the same way across platforms and is easier to reason about.

Good job with the PR 👍

**Comment by Jacse** [comment]
@bgryszko could this be merged?

**Comment by LeonidasEsteban** [comment]
pls @bgryszko :)

**Comment by kenma9123** [comment]
he is ded

**Comment by bgryszko** [comment]
Thanks for a PR!

**Comment by kennydee** [comment]
@bgryszko is it possible to tag a 0.0.9 version on npm with this fix ? Thanks

**Comment by ncuillery** [comment]
>@bgryszko is it possible to tag a 0.0.9 version on npm with this fix ? Thanks

+1 we need this bro

**Comment by abrantes01** [comment]
Any news about npm new tag for this fix ?

**Comment by kennydee** [comment]
Up @bgryszko 
Is it possible to tag a 0.0.9 based on master please ? 
Thanks

**Pull request merged**